CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -417,6 +558,22 @@ template <typename ElementType> class PropertyView<ElementType, false> {
     getNumericPropertyValues(property);
   }
 
+  /**
+   * @brief Constructs a property instance from a property table property, class
+   * definition, and an associated enum definition.
+   */
+  PropertyView(
```

Comment:
Same with the rest of the duplicate constructors in this file. It would be great to consolidate as much as possible.